src/web/nextui/src/app/eval/DownloadMenu.tsx

```diff
@@ -107,41 +179,73 @@ function DownloadMenu() {
+      <Dialog onClose={handleClose} open={open} onKeyDown={handleKeyDown}>
+        <DialogTitle>Download Options</DialogTitle>
         <DialogContent>
           <Stack direction="column" spacing={2} sx={{ width: '100%' }}>
-            <Button
-              onClick={downloadConfig}
-              startIcon={<DownloadIcon />}
-              fullWidth
-              sx={{ justifyContent: 'flex-start' }}
-            >
-              Download YAML Config
-            </Button>
-            <Button
-              onClick={downloadCsv}
-              startIcon={<DownloadIcon />}
-              fullWidth
-              sx={{ justifyContent: 'flex-start' }}
-            >
-              Download Table CSV
-            </Button>
-            <Button
-              onClick={downloadTable}
-              startIcon={<DownloadIcon />}
-              fullWidth
-              sx={{ justifyContent: 'flex-start' }}
-            >
-              Download Table JSON
-            </Button>
-            <Button
-              onClick={downloadDpoJson}
-              startIcon={<DownloadIcon />}
-              fullWidth
-              sx={{ justifyContent: 'flex-start' }}
-            >
-              Download DPO JSON
-            </Button>
+            <Tooltip title="Download the YAML configuration file">
+              <Button
+                onClick={downloadConfig}
+                startIcon={<DownloadIcon />}
+                variant="contained"
+                color="primary"
+                fullWidth
+              >
+                Download YAML Config
+              </Button>
+            </Tooltip>
+
+            <Divider />
+            <Typography variant="subtitle1">Table Data</Typography>
+
+            <Tooltip title="Download table data in CSV format">
+              <Button
+                onClick={downloadCsv}
+                startIcon={<DownloadIcon />}
+                variant="outlined"
+                fullWidth
+              >
+                Download Table CSV
+              </Button>
+            </Tooltip>
+
+            <Tooltip title="Download table data in JSON format">
+              <Button
+                onClick={downloadTable}
+                startIcon={<DownloadIcon />}
+                variant="outlined"
+                fullWidth
+              >
+                Download Table JSON
+              </Button>
+            </Tooltip>
+
+            <Divider />
+            <Typography variant="subtitle1">Advanced Options</Typography>
+
+            <Tooltip title="Download Direct Preference Optimization JSON">
+              <Button
+                onClick={downloadDpoJson}
+                startIcon={<DownloadIcon />}
+                variant="outlined"
+                color="secondary"
+                fullWidth
+              >
+                Download DPO JSON
+              </Button>
+            </Tooltip>
+
+            <Tooltip title="Download Human Evaluation Test Cases in YAML format">
```

Comment:
is "Human Evaluation Test Cases" the right label?  Maybe just Evaluation Test Cases?  or Promptfoo Evaluation Test Cases?